src/battle/battle.c

```diff
@@ -0,0 +1,366 @@
+        .battles = 0x8021F6C8,
+        .stages = 0x8021F72C,
+    },
+    {
+        AREA(area_kzn),
+        .name = "エリア ＫＺＮ",
+        .battles = 0x80229E74,
+        .stages = 0x8022A02C,
+    },
+    {
+        AREA(area_kzn2),
+        .name = "エリア ＫＺＮ２",
+        .battles = 0x8022AA08,
+        .stages = 0x8022AA44,
+        .dmaTable = 0x8022AA84,
+    },
+    {
+        AREA(area_flo),
+        .name = "エリア ＦＬＯ",
+        .battles = 0x80234574,
+        .stages = 0x8023486C,
+    },
+    {
+        AREA(area_flo2),
+        .name = "エリア ＦＬＯ２",
+        .battles = 0x802356FC,
+        .stages = 0x80235788,
+    },
+    {
+        AREA(area_tik),
+        .name = "エリア ＴＩＫ",
+        .battles = 0x802340B4,
+        .stages = 0x802342BC,
+    },
+    {
+        AREA(area_tik2),
+        .name = "エリア ＴＩＫ２",
+        .battles = 0x80227A90,
+        .stages = 0x80227AE0,
+    },
+    {
+        AREA(area_tik3),
+        .name = "エリア ＴＩＫ３",
+        .battles = 0x802232E8,
+        .stages = 0x8022334C,
+    },
+    {
+        AREA(area_sam),
+        .name = "エリア ＳＡＭ",
+        .battles = 0x80231164,
+        .stages = 0x80231290,
+    },
+    {
+        AREA(area_sam2),
+        .name = "エリア ＳＡＭ２",
+        .battles = 0x802234F0,
+        .stages = 0x8022352C,
+    },
+    {
+        AREA(area_pra),
+        .name = "エリア ＰＲＡ",
+        .battles = 0x8022F4D4,
+        .stages = 0x8022F614,
+    },
+    {
+        AREA(area_pra2),
+        .name = "エリア ＰＲＡ２",
+        .battles = 0x802215A0,
+        .stages = 0x802215C8,
+    },
+    {
+        AREA(area_pra3),
+        .name = "エリア ＰＲＡ３",
+        .battles = 0x80232888,
+        .stages = 0x8023298C,
+    },
+    {
+        AREA(area_kpa),
+        .name = "エリア ＫＰＡ",
+        .battles = 0x80231A64,
+        .stages = 0x80231ED8,
+    },
+    {
+        AREA(area_kpa2),
+        .name = "エリア ＫＰＡ２",
+        .battles = 0x802346A8,
+        .stages = 0x80234720,
+    },
+    {
+        AREA(area_kpa3),
+        .name = "エリア ＫＰＡ３",
+        .battles = 0x8022839C,
+        .stages = 0x802283EC,
+    },
+    {
+        AREA(area_kpa4),
+        .name = "エリア ＫＰＡ４",
+        .battles = 0x80227910,
+        .stages = 0x80227988,
+    },
+    {
+        AREA(area_kkj),
+        .name = "エリア ＫＫＪ",
+        .battles = 0x802185BC,
+        .stages = 0x802185E4,
+    },
+    {
+        AREA(area_dig),
+        .name = "エリア ＤＩＧ",
+        .battles = 0x8022A7C4,
+        .stages = 0x8022A83C,
+    },
+    {},
+    {
+        AREA(area_omo2_1),
+        .name = "エリア ＯＭＯ２＿１",
+    },
+    {
+        AREA(area_omo2_2),
+        .name = "エリア ＯＭＯ２＿２",
+    },
+    {
+        AREA(area_omo2_3),
+        .name = "エリア ＯＭＯ２＿３",
+    },
+    {
+        AREA(area_omo2_4),
+        .name = "エリア ＯＭＯ２＿４",
+    },
+    {
+        AREA(area_omo2_5),
+        .name = "エリア ＯＭＯ２＿５",
+    },
+    {
+        AREA(area_omo2_6),
+        .name = "エリア ＯＭＯ２＿６",
+    },
+    {},
+};
+
+const s32 rodata_75978[] = {
```

Comment:
hm. Is it possible to put these in a rodata.s file, since this is most likely a jump table, and the data would never be defined in C?

Reply:
Possibly, I have no idea how to do that though

Reply:
It'd help to have the vram addresses of these rodata_ vars and a comment saying it's most likely a jump table. Beyond that, I think we can merge this as-is and migrate the rodata later on, in the interest of keeping things moving

Reply:
I added the comment